ban/http/api.py

```diff
@@ -350,7 +350,7 @@ def get_version(self, identifier, ref):
         instance = self.get_object(identifier)
         version = instance.load_version(ref)
         if not version:
-            abort(404)
+            abort(404, error='Version reference `{}` not found'.format(ref))
```

Comment:
In this case I would use double tick for assigning the variable and single quotes for inner string. Just an opinion.